### PR TITLE
{toolchain}[dummy/dummy] goolfc v2017b

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
New version of goolfc toolchain. Uses same versions of dependencies as foss/2017b but adds CUDA 9.0

### CODE REVIEW

File: easybuild/easyconfigs/c/CUDA/CUDA-9.0.176-GCC-6.4.0-2.28.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+
+source_urls = [
+    'https://developer.nvidia.com/compute/cuda/%(version_major_minor)s/prod/local_installers/',
+    'http://developer.download.nvidia.com/compute/cuda/%(version_major_minor)s/Prod/local_installers/',
+    'https://developer.nvidia.com/compute/cuda/%(version_major_minor)s/Prod2/local_installers/',
+    'https://developer.nvidia.com/compute/%(namelower)s/%(version_major_minor)s/Prod/local_installers/',
+]
+sources = ['%%(namelower)s_%%(version)s_%s_linux-run' % nv_version]
+#checksums = ['8496c72b16fee61889f9281449b5d633d0b358b46579175c275d85c9205fe953']
```
Comment (boegel) [inline easybuild/easyconfigs/c/CUDA/CUDA-9.0.176-GCC-6.4.0-2.28.eb]: @akesandgren Please fix? I have `96863423feaa50b5c1c5e1b9ec537ef7ba77576a3986652351ae43e66bcd080c` as SHA256 for `cuda_9.0.176_384.81_linux-run`, this one you have here is the one for `cuda_9.1.85_387.26_linux`

Comment (akesandgren) [reply easybuild/easyconfigs/c/CUDA/CUDA-9.0.176-GCC-6.4.0-2.28.eb]: Oops

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2085.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9e6f0d5f30d73395b8f35d8b6b0a0972 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2567.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/674321b1804a0b39f4791abdff153f40 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Comment by wpoely86** [comment]
```
libtool:   error: require no space between '-L' and '-lutil'
```

That's new for me.

**Comment by wpoely86** [comment]
In `openmpi-2.1.1/ompi/mca/pml/ucx/Makefile` I see:
```
atomic_ucx_LIBS =  -lucp -L
```

Not sure what is supposed to be there.

**Comment by akesandgren** [comment]
I have
atomic_ucx_LIBS = 
so you'll have to figure out where it comes from

**Comment by boegel** [comment]
@wpoely86 Have you figured this out yet?

**Comment by akesandgren** [comment]
Ah, it's the UCX framework? Is that installed as a OS package?
If so there might be a bug in OpenMPI if it shows just -L

**Comment by wpoely86** [comment]
Yes, removing the rpms for UCX fixed it. It seems the configure script fails to do it correctly.

**Pull request merged**